websocket_server/websocket_server.py

```diff
@@ -108,21 +108,34 @@ class WebsocketServer(ThreadingMixIn, TCPServer, API):
                 {
                  'id'      : id,
                  'handler' : handler,
-                 'address' : (addr, port)
+                 'address' : (addr, port),
+		 'server'  : websocket instance
```

Comment:
Also should not be required. Simply the new behaviour should be for multicast to send to clients of the instance